src/web/templates/components/parameters.html

```diff
@@ -30,21 +30,24 @@
   }
 
   function renderLayout() {
-    showLoading("Rendering layout...")
+    showLoading("Rendering Layout...")
     disallowRunSimulation()
     clearPlotArea()
 
+    const is2d = scenarioIs2d()
     const scenarioParams = getScenarioParams()
     const transducersParams = getTransducersParams()
     const targetParams = getTargetParams()
     const simulationSettingsParams = getSimulationSettingsParams()
+    const displaySettingsParams = is2d ? null : getDisplaySettingsParams()
 
     const data = {
-      is2d: scenarioIs2d(),
+      is2d,
```

Comment:
Should this be `is2d: is2d`?
(I don't know much Javascript)

Reply:
this is a shorthand when the key name is the same as the variable name:
https://www.geeksforgeeks.org/shorthand-syntax-for-object-property-value-in-es6/